tests/unit/test_help.py

```diff
@@ -385,8 +415,9 @@ def test_command_help_text_no_parameters(output_format):
     assert text == (expected_plain if output_format == OutputFormat.plain else expected_markdown)
 
 
+@pytest.mark.parametrize("docs_url", [None, "www.craft-app.com/docs/3.14159/"])
 @pytest.mark.parametrize("output_format", list(OutputFormat))
-def test_command_help_text_with_parameters(output_format):
+def test_command_help_text_with_parameters(docs_url, output_format):
```

Comment:
You could make a parametrized fixture, so you don't have to parametrize every test separately.

```python
@pytext.fixture(params=[None, "www.craft-app.com/docs/3.14159/"]):
def docs_url(request) -> str | None:
    return request.param
```